pressbooks/pressbooks
# About the authors:

Related issue: https://github.com/pressbooks/pressbooks/issues/2347

This PR adds the **About the Authors** section at the end of each chapter to the HTMLBook export routine.
It also remove the contributors picture size validation at the backend level in order to avoid the validation in other media uploading.
Now the validation is done using the wp media library as recommended by WP.

### Test
- For a random book, assign different authors to a different chapters. Make sure those authors are book's contributor in the Book Info section.
- Make sure Appearance > Theme Options > About the authors options is enabled.
- To test it better, you may add picture, url, description and/or other fields to the contributors in Book Info > Contributors section.
- Export the book to HTMLBook and make sure the authors are added at the end of those chapter with authors. Take into account styling is not supported for this format.
